docs/self-managed/zeebe-deployment/operations/update-zeebe.md

```diff
@@ -163,6 +163,10 @@ If the error is caused by `Cannot upgrade to or from a pre-release version`, Zee
 This is not permitted because pre-release versions such as alpha releases are considered unstable and do not guarantee compatibility with any other version.
 If you attempted to update from a normal version to a pre-release version, you can roll back to the previous normal version.
 
+:::note
+While in most instances version rollbacks are not supported, in this particular use case a version rollback is possible.
```

Comment:
```suggestion
While in most instances version rollbacks are not supported, in this particular use case a version rollback of Zeebe is possible because the "new" version never became active, thanks to the safety check.
```

Reply:
I'm not sure the extra explanation adds a lot here. We don't otherwise talk about what the safety checks or doing, and can possibly introduce new points of confusion re: "new" vs "old" instead of clarifying. If we really think our developers need this info, I'd suggest something like: 

`..a version rollback of Zeebe is possible, as the included safety checks prevented the pre-release version from becoming active.`

Reply:
I'd keep the addition of "Zeebe" then because rollback of the other applications might not be possible.

Reply:
Updated in latest commit -- let me know what you think.